os/mm/umm_heap/umm_heap_malloc.c

```diff
@@ -0,0 +1,56 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/************************************************************************
+ * Included Files
+ ************************************************************************/
+
+#include <tinyara/config.h>
+#include <tinyara/mm/mm.h>
+
+/************************************************************************
+ * Name: umm_heap_malloc
+ *
+ * Description:
+ *   This is a simple wrapper for the mm_malloc() function.
+ *   This function is used to allocate the memory from kernel side
+ *   to specified user heap.
+ *
+ * Parameters:
+ *   heap - Which user heap to allocate
+ *   size  - The size (in bytes)
+ *
+ * Return Value:
+ *   Allocated address on success, NULL on failure.
```

Comment:
It might be the starting address of assigned memory, right?

Reply:
Yes, right. I will modify the description. Thanks.